tcms/core/tests/test_views.py

```diff
@@ -83,3 +86,27 @@ def test_dashboard_shows_testruns_for_execution_assignee(self):
 
         response = self.client.get(reverse('core-views-index'))
         self.assertContains(response, execution.run.summary)
+
+
+def exception_view(request):
+    raise Exception
+
+
+urlpatterns = [
+    path('500/', exception_view),
+    path('', include(urls)),
+]
+
+
+handler500 = 'tcms.core.views.server_error'
+
+
+@override_settings(ROOT_URLCONF=__name__)
+class TestServerError(TestCase):
+
+    def test_custom_server_error_view(self):
+        client = Client(raise_request_exception=False)
```

Comment:
The reason for using `raise_request_exception` : https://code.djangoproject.com/ticket/18707
When using `self.client.get('/500/')`, Django wasn't returning the `HttpResponseServerError` instead it was failing with a exception.